business_objects/information_source.py

```diff
@@ -168,7 +179,7 @@ def get_exclusion_record_ids_for_task(task_id: str) -> List[str]:
     return exclusion_ids
 
 
-def get_overview_data(project_id: str) -> List[Dict[str, Any]]:
+def get_overview_data(project_id: str, type_selection: str) -> List[Dict[str, Any]]:
```

Comment:
sorry what i meant was the boolean value and evaluation here so someone using the function knows what to do without checking the SQL code

Reply:
makes sense